core/src/main/java/sunstone/core/SunstoneConfig.java

```diff
@@ -0,0 +1,79 @@
+package sunstone.core;
+
+
+import io.smallrye.config.SmallRyeConfig;
+import io.smallrye.config.SmallRyeConfigBuilder;
+import org.eclipse.microprofile.config.spi.ConfigSource;
+
+import java.util.Objects;
+import java.util.Set;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.ConcurrentMap;
+
+public class SunstoneConfig {
```

Comment:
Isn't existence of such class indication that it is cleaner to have `sunstone.properties` ? If we could need some logic around standard?